build.sh

```diff
@@ -46,7 +46,7 @@ if [[ $OSTYPE == darwin* ]] ; then
     PARALLEL=$(sysctl -n hw.ncpu)
     # We know for sure that build-thirdparty.sh will fail on darwin platform, so just skip the step.
 else
-    if [[ ! -f ${STARROCKS_THIRDPARTY}/installed/include/datasketches/hll.hpp ]]; then
+    if [[ ! -f ${STARROCKS_THIRDPARTY}/installed/async-profiler/profiler.sh ]]; then
```

Comment:
this is not a must have change to the compiling, better not change the condition to disturb everyone for a thirdparty rebuild.

Reply:
done